src/Add.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Laminas\Filter;
+
+use Laminas\Filter\AbstractFilter;
+use Laminas\Filter\Exception\InvalidArgumentException;
+
+class Add extends AbstractArithmeticOperation
+{
+    /**
+     *
+     * {@inheritdoc}
+     * @see \Laminas\Filter\FilterInterface::filter()
+     */
+    public function filter($value)
+    {
+        if (! isset($this->options['operand'])) {
+            throw new InvalidArgumentException(sprintf('%s expects argument value; received none', __METHOD__));
+        }
+
+        $value = (float) $value;
+        $operand = (float) $this->options['operand'];
```

Comment:
Why the cast to float here?

If I were operating on integers, I'd expect an integer return value. Perhaps this should be broken into "AddInteger" and "AddFloat"?